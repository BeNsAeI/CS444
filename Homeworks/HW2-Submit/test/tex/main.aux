\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0 << /S /D >> }
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Look}{3}{section.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Look ahead}{3}{subsection.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}Bias}{3}{subsection.1.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3}Complications}{3}{subsection.1.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.4}Variants}{3}{subsection.1.4}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.4.1}C-Look}{3}{subsubsection.1.4.1}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Illustration of Circular Look ahead algorithm for scheduler}}{4}{table.1}}
\newlabel{C-Look}{{1}{4}{Illustration of Circular Look ahead algorithm for scheduler\relax }{table.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.4.2} F-Look, N-Look}{4}{subsubsection.1.4.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.4.3}S-Look}{4}{subsubsection.1.4.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.5}Our Implementation: C-Look}{4}{subsection.1.5}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.6}Code}{5}{subsection.1.6}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Questions}{7}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Response to Homework Questions}{7}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1}What do you think the main point of this assignment is?}{7}{subsubsection.2.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.2}How did you personally approach the problem? Design decisions, algorithm, etc.}{8}{subsubsection.2.1.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.3}How did you ensure your solution was correct? Testing details, for instance.}{8}{subsubsection.2.1.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Graphing disk sector access during IO usage}}{9}{figure.1}}
\newlabel{iograph}{{1}{9}{Graphing disk sector access during IO usage\relax }{figure.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.4}What did you learn?}{9}{subsubsection.2.1.4}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Dining Philosopher}{9}{section.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}Problem}{9}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}Solution}{10}{subsection.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.3}Application}{10}{subsection.3.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.4}Implementation}{10}{subsection.3.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.5}Code}{10}{subsection.3.5}}
\gdef\minted@oldcachelist{,
  default-pyg-prefix.pygstyle,
  default.pygstyle,
  45FCB48DF4C38EE1A59D9BCC649F0C5C5093E84BEA004B2911D4DA186861F6DB.pygtex,
  C29A2AE28E09353A0EC0F14FC81253A1042D2615758B6C3F03C186B02CFC62B2.pygtex,
  B8E3877E53C8E99E0BFC4D7EE4638FB04FBEEAFC55D0059826E664B9BBCF3721.pygtex,
  15D43D3345B28AC081A128FDF6387492042D2615758B6C3F03C186B02CFC62B2.pygtex}
\@writefile{toc}{\contentsline {section}{\numberline {4}Work Log}{12}{section.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}Hours}{12}{subsection.4.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.2}Git Log}{12}{subsection.4.2}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces Git log for this assignment}}{13}{table.2}}
\newlabel{GitLog}{{2}{13}{Git log for this assignment\relax }{table.2}{}}
