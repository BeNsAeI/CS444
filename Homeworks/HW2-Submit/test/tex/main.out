\BOOKMARK [1][-]{section.1}{Look}{}
\BOOKMARK [2][-]{subsection.1.1}{Look ahead}{section.1}
\BOOKMARK [2][-]{subsection.1.2}{Bias}{section.1}
\BOOKMARK [2][-]{subsection.1.3}{Complications}{section.1}
\BOOKMARK [2][-]{subsection.1.4}{Variants}{section.1}
\BOOKMARK [3][-]{subsubsection.1.4.1}{C-Look}{subsection.1.4}
\BOOKMARK [3][-]{subsubsection.1.4.2}{ F-Look, N-Look}{subsection.1.4}
\BOOKMARK [3][-]{subsubsection.1.4.3}{S-Look}{subsection.1.4}
\BOOKMARK [2][-]{subsection.1.5}{Our Implementation: C-Look}{section.1}
\BOOKMARK [2][-]{subsection.1.6}{Code}{section.1}
\BOOKMARK [1][-]{section.2}{Questions}{}
\BOOKMARK [2][-]{subsection.2.1}{Response to Homework Questions}{section.2}
\BOOKMARK [3][-]{subsubsection.2.1.1}{What do you think the main point of this assignment is?}{subsection.2.1}
\BOOKMARK [3][-]{subsubsection.2.1.2}{How did you personally approach the problem? Design decisions, algorithm, etc.}{subsection.2.1}
\BOOKMARK [3][-]{subsubsection.2.1.3}{How did you ensure your solution was correct? Testing details, for instance.}{subsection.2.1}
\BOOKMARK [3][-]{subsubsection.2.1.4}{What did you learn?}{subsection.2.1}
\BOOKMARK [1][-]{section.3}{Dining Philosopher}{}
\BOOKMARK [2][-]{subsection.3.1}{Problem}{section.3}
\BOOKMARK [2][-]{subsection.3.2}{Solution}{section.3}
\BOOKMARK [2][-]{subsection.3.3}{Application}{section.3}
\BOOKMARK [2][-]{subsection.3.4}{Implementation}{section.3}
\BOOKMARK [2][-]{subsection.3.5}{Code}{section.3}
\BOOKMARK [1][-]{section.4}{Work Log}{}
\BOOKMARK [2][-]{subsection.4.1}{Hours}{section.4}
\BOOKMARK [2][-]{subsection.4.2}{Git Log}{section.4}
